test/src/test/testtool/FilePipe/common.properties

```diff
@@ -7,26 +7,26 @@ java.FilePipe.className=nl.nn.adapterframework.senders.IbisJavaSender
 java.FilePipe.serviceName=ibis4test-FilePipe
 
 fs.insert.className=nl.nn.adapterframework.testtool.FileSender
-fs.insert.filename=${testdata.dir}/fp2/dummy.txt
+fs.insert.filename=${testdata.dir}/dummy.txt
```

Comment:
Why did you remove the subfolders? Without it will be more difficult to maintain the code and see what file/folder belongs where...

Reply:
Because there is nothing that creates fp2/ folder therefor starting iaf-test will result in one adapter/configuration that will not start properly. Also, I had previously created a DirectoryListener for that adapter/configuration but Gerrit didn't approve of that solution. So we decided on changing the folder where files are created.